Documentation/Hardware.md

```diff
@@ -12,23 +33,32 @@ TS100\* is a neat soldering iron:
 
 ![](https://brushlesswhoop.com/images/ts100-og.jpg)
 
+### TS101
+
+The TS101 is the direct replacement of the TS100 with the same tip compatibility.
+It adds a spring pressure tip holding mechanism instead of using a screw so tips are easier to swap on the fly (But are held less securely and can pull out depending on the use case). It adds USB-C PD support and the hardware is compatible with 28V EPR power supplies (under both IronOS and official firmware).
+
+It unfortunately uses an STM32 clone MCU with quirks, so performance of the screen isn't as good as it could be but its perfectly usable. The bootloader for programming is the biggest weakness of this device and programming can be a pain. Thankfully IronOS is relatively stable feature wise so you shouldn't need to update the device especially often.
```

Comment:
```suggestion
It unfortunately uses an STM32 clone MCU with quirks, so performance of the screen isn't as good as it could be but its perfectly usable. The bootloader for programming is the biggest weakness of this device and programming can be a pain. Fortunately, IronOS is relatively stable feature wise, so you shouldn't need to update the device often.
```